README.md

```diff
@@ -21,6 +21,18 @@ Inside the project, install the project's dependencies:
 cd igbo_api/
 yarn install
 ```
+If you are running the API on windows Operating system, run the module to install it globally:
+
+```
+cd igbo_api/                   
+npm install -g win-node-env
+```
+Or run the module to install it In this particular project:
```

Comment:
I don't think we should locally add environment-specific modules in the project. This section can be removed

Reply:
okay